10-JavaScript/arrow-functions-anonymous-this.md

```diff
@@ -0,0 +1,136 @@
+# Arrow Functions: as Anonymous Functions, and `this`
+
+## Learning Goals
+
+- Recognize using arrow functions as anonymous functions
+- Practice refactoring anonymous functions as arrow functions
```

Comment:
Much of the rest of our curriculum formats these as "By the end of this lesson, students will be able to...". I've also heard that's an important framing technique, to make sure your learning goals are student-focused.